static/docs/commands-reference/version.md

```diff
@@ -14,18 +14,19 @@ usage: dvc version [-h] [-q | -v]
 system/environment:
 
-| Type              | Detail                                                                                                                                                    |
-| ----------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `DVC version`     | Version of DVC (along with a Git commit hash in case of a development version)                                                                            |
-| `Python version`  | Version of the Python being used for the project in which DVC is initialized                                                                              |
-| `Platform`        | Information about the operating system of the machine                                                                                                     |
-| `Cache`           | [Type of links](/doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache) supported between the DVC workspace and the cache directory |
-| `Filesystem type` | Shows the filesystem type (eg. ext4, FAT, etc.) and mount point of workspace and the cache directory                                                      |
+| Type                 | Detail                                                                                                                                                    |
+| -------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [`DVC version`](#vc) | Version of DVC (along with a Git commit hash in case of a development version)                                                                            |
+| `Python version`     | Version of the Python being used for the project in which DVC is initialized                                                                              |
+| `Platform`           | Information about the operating system of the machine                                                                                                     |
+| [`Binary`](#bin)     | Shows whether the package is installed from a binary release or source                                                                                    |
+| `Cache`              | [Type of links](/doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache) supported between the DVC workspace and the cache directory |
+| `Filesystem type`    | Shows the filesystem type (eg. ext4, FAT, etc.) and mount point of workspace and the cache directory                                                      |
 
 > If `dvc version` is executed outside a DVC workspace, the command outputs the
 > filesystem type of the current working directory.
 
-#### Components of DVC version
+#### <a name="vc">Components of DVC version</a>
```

Comment:
you don't need <a> here, it generates an anchor automatically - #components-of-dvc-version can be used